packages/runtime/src/components/builtin/Button/Button.tsx

```diff
@@ -27,12 +26,13 @@ import {
   useResponsiveWidth,
 } from '../../utils/responsive-style'
 import { cx } from '@emotion/css'
+import { LinkPropControllerValue } from '@makeswift/prop-controllers'
 
 type BaseProps<T extends ElementType> = {
   as?: T
   id?: ElementIDValue
   children?: ReactNode
-  link?: LinkValue
+  link?: LinkPropControllerValue
```

Comment:
We should export a generic `Link` type which is used by multiple prop controllers.